bin/install

```diff
@@ -69,4 +69,21 @@ install_rust() {
   )
 }
 
+install_default_rust_packages() {
+  local packages_file="${ASDF_RUST_DEFAULT_PACKAGES_FILE:-$HOME/.default-rust-packages}"
+
+  if [ -f "$packages_file" ]; then
+    echo -ne "\nInstalling default rust packages..."
+    while read -r package; do
+      echo -ne "\nInstalling rust package $package..."
+      echo "Pouet : $ASDF_INSTALL_PATH"
+      PATH="$ASDF_INSTALL_PATH/bin:$PATH" cargo install -f --locked "$package" --root "$ASDF_INSTALL_PATH/.cargo"
+    done <"$packages_file"
+    for binary in "$ASDF_INSTALL_PATH/.cargo/bin"/*; do
+      ln -s "$binary" "$ASDF_INSTALL_PATH/bin/$(basename "$binary")"
+    done
```

Comment:
With https://github.com/asdf-community/asdf-rust/pull/32 , the cargo packages are installed directly into this folder, so no need to pass `--root`, or symlinking the binaries :wink:

Reply:
Good to know, but depends on this PR to be merged, or cherry-picking the modifications or this PR ;)